src/main/java/digital/slovensko/autogram/core/UserSettings.java

```diff
@@ -55,9 +55,9 @@ public static UserSettings load() {
         settings.setPdfaCompliance(prefs.getBoolean("PDFA_COMPLIANCE", true));
         settings.setServerEnabled(prefs.getBoolean("SERVER_ENABLED", true));
         settings.setExpiredCertsEnabled(prefs.getBoolean("EXPIRED_CERTS_ENABLED", false));
-        settings.setTrustedList(prefs.get("TRUSTED_LIST", "SK,CZ,AT,PL,HU,BE,NL,BG"));
+        settings.setTrustedList(prefs.get("TRUSTED_LIST", "SK,CZ,AT,PL,HU,BE,NL,LT"));
         settings.setCustomKeystorePath(prefs.get("CUSTOM_KEYSTORE_PATH", ""));
-        settings.setTsaServer(prefs.get("TSA_SERVER", "http://tsa.belgium.be/connect,http://ts.quovadisglobal.com/eu,http://tsa.sep.bg"));
+        settings.setTsaServer(prefs.get("TSA_SERVER", "http://ts.quovadisglobal.com/eu,http://tsa.baltstamp.lt"));
```

Comment:
ešte tu by sme vedlei spraviť takú vec, že keď sa tá hodnota rovná `http://tsa.belgium.be/connect,http://ts.quovadisglobal.com/eu,http://tsa.sep.bg` (čiže starý default), tak by sme ju nahradili novým default.

Reply:
ako forcnúť na zoznam? Ak to bude presne ten string, tak to znamená, že user nechal nastavený default. A asi aj chce mať default aj ďalej, nech už je hociaký.

Reply:
no minimalne bude divne, ze nastavis zoznam nejako a on sa nejako pod rukami zmeni. Ale toto je asi specialitka co bude len docasne.

Ono by davalo skor zmysel tie defaulty drzat na nejaky null a ked to je null tak sa taha default z distribucie, nie nastavit nieco.